src/rulesets/Base/base.jl

```diff
@@ -119,6 +119,40 @@ function rrule(::typeof(hypot), z::Complex)
     return (Ω, hypot_pullback)
 end
 
+# Note that `hypot` with two arguments has rules in `fastmath_able.jl`
+
+function frule(
+    (_, Δx, Δy, Δz, Δxs...),
+    ::typeof(hypot),
+    x::T,
+    y::T,
+    z::T,
+    xs::Vararg{T, N}
+) where {T<:Union{Real,Complex}, N}
+    Ω = hypot(x, y, z, xs...)
+    n = ifelse(iszero(Ω), one(Ω), Ω)
```

Comment:
Generally, `one(x)` is not of the same type as `x`:
```suggestion
    n = ifelse(iszero(Ω), oneunit(Ω), Ω)
```